screens/FeedScreen.tsx

```diff
@@ -306,48 +312,51 @@ export default function FeedScreen({
   ]);
 
   return (
-    <APIList
-      {...rest}
-      ref={listRef}
-      style={[{ flex: 1 }, style]}
-      viewabilityConfigCallbackPairs={viewabilityConfigCallbackPairs.current}
-      ListRef={scrollRef}
-      ListFooterComponent={footerComponent}
-      ListHeaderComponent={
-        <View style={{}}>
-          <KeyboardAvoidingView
-            enabled={rest.inverted ?? false}
-            behavior={Platform.OS === "ios" ? "position" : "height"}
-            keyboardVerticalOffset={90}
-            style={{ flex: 1 }}
-          >
-            <PostInputField
-              onPostAdded={(newPost: Post) => {
-                setAddedNewPost(true);
-                listRef.current?.addItem({...newPost, 
-              userId: globalThis.myId,
-              createdAt: "null",
-            }); onPostAdded?.(newPost);}}
-              channel={channel}
-              originalParentId={originalParentId}
-              autoFocus={autoFocus}
-              isChallenge={isChallenge}
-              label={postButtonLabel}
-            />
-          </KeyboardAvoidingView>
-          {headerComponent}
-        </View>
-      }
-      initialAmount={7}
-      additionalAmount={7} //change number based on device specs
-      processingFunction={getLikedPosts as any}
-      queryOperation={isChallenge ? postsByLikes : postsByChannel}
-      queryOperationName={isChallenge ? "postsByLikes" : "postsByChannel"}
-      filter={{ channel: channel, sortDirection: "DESC" }}
-      renderItem={renderPostItem}
-      keyExtractor={(item: Post) => item.createdAt.toString() + item.userId}
-      onEndReachedThreshold={0.5}
-    />
+    <>
+      <NearbyActivities />
```

Comment:
We can exclude this from the PR, I think Matthew had plans to rework the feed screen. You can keep these changes on your local though for testing.